lib/galaxy/tools/__init__.py

```diff
@@ -401,6 +402,17 @@ def allow_user_access( self, user, attempting_access=True ):
         """
         return True
 
+    def _get_tours(self):
+        ret = []
+        # ToDo: only for TS tools
+        # if self.repository_id:
+        tours_dir = os.path.join(self.tool_dir,'tours')
```

Comment:
Here we probably want to filter only for TS tools or we need to implement an mechanism to map one tours.yaml file to one tool.
Currently this is intended for single tools repositories (we are aiming for that as IUC and with planemo demultiplex).